config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,15 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+#: --tmpfs parameter
+tmpfs_path = /run
+run_options_csv = --tmpfs,%(tmpfs_path)s
+#: command to run in container to check if passed --tmpfs is mounted as tmpfs type
+fndmnt = findmnt -n -t tmpfs %(tmpfs_path)s
+cmd = '%(fndmnt)s'
```

Comment:
My 'why sleep' was an earnest question: I didn't know if it was necessary or just for luck.

If it isn't necessary, though, neither is this two-step process: there's no need to define an intermediate single-purpose variable, it's much cleaner to just say:

    cmd = findmnt ...